### ISSUE TITLE
watchman asked for calendar and contact permissions on macOS 10.14

### ISSUE THREAD

**Issue by MoOx** [facebook/watchman#616]
First time I opened my terminal since I updated to 10.14, I got this

![image](https://user-images.githubusercontent.com/157534/41022699-a4160eec-696a-11e8-9b50-9c48fbafcc8c.png)

![image](https://user-images.githubusercontent.com/157534/41022706-a935d3c6-696a-11e8-9d0d-ada6b32cbaf8.png)

Does not make sense for what watchman does.

**Comment by sunshowers** [comment]
Watchman definitely doesn't request those permissions, unless there's some weird launchd-related configuration issue. Are you sure it's this project?

**Comment by wez** [comment]
There's a module on npm that also installs a watchman binary.  Perhaps that is what you have installed instead?

**Comment by MoOx** [comment]
I installed watchman via brew a while ago. Nothing more. I upgraded to Mojave and got this as soon as I opened a terminal.

But I just got this when doing a find 

<img width="420" alt="image" src="https://user-images.githubusercontent.com/157534/41091521-05290d42-6a47-11e8-85fd-1009c8aa46d0.png">

(and same for calendar right after)

So this is probably not directly related to watchman, but it looks like it. I guess that's a iTerm issue somehow...
